### ISSUE TITLE
[0.6.19][PHPSTORM 2023.3 Beta] com.intellij.diagnostic.PluginException: 438 ms to call on EDT CreateCommitMessageActi...

### ISSUE THREAD

**Issue by pandiya-nithra** [Codium-ai/codiumai-jetbrains-release#149]
# User Description
_User did not enter any detailed information._

# Stack Trace
```
com.intellij.diagnostic.PluginException: 438 ms to call on EDT CreateCommitMessageAction#Update@ChangesView.CommitToolbar (ai.codium.ui.actions.CreateCommitMessageAction). Revise AnAction.getActionUpdateThread property [Plugin: codiumai.codiumai]
	at com.intellij.diagnostic.PluginProblemReporterImpl.createPluginExceptionByClass(PluginProblemReporterImpl.java:23)
	at com.intellij.diagnostic.PluginException.createByClass(PluginException.java:89)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater.computeOnEdt(ActionUpdater.kt:225)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater.access$computeOnEdt(ActionUpdater.kt:73)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater$computeOnEdt$1.invokeSuspend(ActionUpdater.kt)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:108)
	at kotlinx.coroutines.internal.LimitedDispatcher$Worker.run(LimitedDispatcher.kt:115)
	at kotlinx.coroutines.scheduling.TaskImpl.run(Tasks.kt:103)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:584)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:793)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:697)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:684)
	Suppressed: java.lang.Throwable: EDT-trace-at-101-ms (1/5)
		at java.base@17.0.9/java.security.AccessController.getStackAccessControlContext(Native Method)
		at java.base@17.0.9/java.security.AccessController.getContext(AccessController.java:1003)
		at java.desktop/java.awt.Component.<init>(Component.java:512)
		at java.desktop/java.awt.Container.<init>(Container.java:300)
		at java.desktop/javax.swing.JComponent.<init>(JComponent.java:641)
		at java.desktop/javax.swing.JPanel.<init>(JPanel.java:85)
		at java.desktop/javax.swing.JPanel.<init>(JPanel.java:112)
		at java.desktop/javax.swing.JPanel.<init>(JPanel.java:120)
		at com.intellij.ui.CaptionPanel.<init>(CaptionPanel.java:25)
		at com.intellij.ui.TitlePanel.<init>(TitlePanel.java:32)
		at com.intellij.ui.TitlePanel.<init>(TitlePanel.java:29)
		at com.intellij.openapi.progress.util.ProgressDialogUI.<init>(ProgressDialogUI.kt:33)
		at com.intellij.openapi.progress.util.ProgressDialog.<init>(ProgressDialog.kt:43)
		at com.intellij.openapi.progress.util.ProgressWindow.lambda$new$0(ProgressWindow.java:103)
		at com.intellij.openapi.progress.util.ProgressWindow$$Lambda$2440/0x0000000801516000.run(Unknown Source)
		at com.intellij.openapi.progress.util.ProgressWindow.initializeDialog(ProgressWindow.java:123)

<TRIMMED STACKTRACE>

733)
		at java.base@17.0.9/java.security.AccessController.executePrivileged(AccessController.java:776)
		at java.base@17.0.9/java.security.AccessController.doPrivileged(AccessController.java:399)
		at java.base@17.0.9/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
		at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:761)
		at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.kt:695)
		at com.intellij.ide.IdeEventQueue._dispatchEvent$lambda$12(IdeEventQueue.kt:589)
		at com.intellij.ide.IdeEventQueue$$Lambda$720/0x0000000800774000.run(Unknown Source)
		at com.intellij.openapi.application.impl.RwLockHolder.runWithoutImplicitRead(RwLockHolder.kt:44)
		at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.kt:589)
		at com.intellij.ide.IdeEventQueue.access$_dispatchEvent(IdeEventQueue.kt:72)
		at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1$1.compute(IdeEventQueue.kt:355)
		at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1$1.compute(IdeEventQueue.kt:354)
		at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:793)
		at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1.invoke(IdeEventQueue.kt:354)
		at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1.invoke(IdeEventQueue.kt:349)
		at com.intellij.ide.IdeEventQueueKt.performActivity$lambda$1(IdeEventQueue.kt:1014)
		at com.intellij.ide.IdeEventQueueKt$$Lambda$719/0x000000080076f410.run(Unknown Source)
		at com.intellij.openapi.application.TransactionGuardImpl.performActivity(TransactionGuardImpl.java:106)
		at com.intellij.ide.IdeEventQueueKt.performActivity(IdeEventQueue.kt:1014)
		at com.intellij.ide.IdeEventQueue.dispatchEvent$lambda$7(IdeEventQueue.kt:349)
		at com.intellij.ide.IdeEventQueue$$Lambda$717/0x000000080076a5e0.run(Unknown Source)
		at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:848)
		at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.kt:391)
		at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:207)
		at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:128)
		at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:117)
		at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:113)
		at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:105)
		at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:92)

```

# Other Information
```
=== About ===
Build version: PhpStorm 2023.3 Beta Build: #PS-233.11799.21 November 13, 2023
Theme: Dark
JRE: 17.0.9+7-b1087.3, JetBrains s.r.o.
JVM: 17.0.9+7-b1087.3, OpenJDK 64-Bit Server VM, JetBrains s.r.o.
Operating System: Windows 11.0 (amd64)

=== Plugins ===
Custom plugins: [PHP Annotations (9.4.0), CodiumAI - Integrity Agent powered by GPT-3.5&4 (0.6.19), Laravel Tinker (2.6.1), FileSync (1.15.0), .env files support (2023.3), Elm (4.4.1), PHP Advanced AutoComplete (1.1.0), Python custom import (2.0.2), AI Assistant (233.11799.38), CodeGPT - ChatGPT For Laravel (1.05), GitHub Copilot (1.4.2.3864), Copilot Word by Word (0.1.0), PHP Toolbox (6.2.0), Space (233.11799.29), Laravel Query (4.0.2), SQL Optimizer, Indexing Advisor, MySQL, PostgreSQL, by EverSQL (1.3.1), Php Inspections (EA Extended) (5.0.0.0)]
Disabled plugins:[Copilot Dark Theme (1.0.4), PHPUnit code coverage (0.5.0), ChatGPT X-Copilot Context Aware & Function calling (2023.10.21)]
```

**Comment by ArtsiomCh** [comment]
Thanks for reporting! We are working on it.

**Comment by ArtsiomCh** [comment]
Will be fixed in 0.7.05
Thanks again for reporting!
